### ISSUE TITLE
Error updating package Microsoft.NET.Test.Sdk in project App1.UWP.Test

### ISSUE THREAD

**Issue by romeop555** [Plac3hold3r/MvxScaffolding#536]
Reproduce the issue:
- Add a new Forms project Navigation type with UWP in VS2017 with the name of App1
- try to update the package with the command "Install-Package Microsoft.NET.Test.Sdk -ProjectName App1.UWP.Test"

Log:
Restoring packages for C:\Users\romeo\source\repos\App1\test\App1.UWP.Test\App1.UWP.Test.csproj...
Install-Package : The system cannot find the file specified.
At line:1 char:1
+ Install-Package Microsoft.NET.Test.Sdk -ProjectName App1.UWP.Test
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Install-Package], Exception
    + FullyQualifiedErrorId : NuGetCmdletUnhandledException,NuGet.PackageManagement.PowerShellCmdlets.InstallPackageCommand
 
Time Elapsed: 00:00:05.1343707

**Comment by Plac3hold3r** [comment]
Thanks @romeop555 , I believe I finally figured this out, why this is happening. 

So the default Visual Studio UWP test template (which the MvxScaffolding UWP test template is base on) makes use of the reference to the `Microsoft.NET.Test.Sdk` installed with Visual Studio. You can find this reference under `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\WindowsXaml\v16.0\Microsoft.Windows.UI.Xaml.Common.Targets` for the PackageReference version. This prevents the NuGet package from being updated and let Visual Studio control the allowed package version. 

Removing the following from the csproj and installing the NuGet package directly will allow for updating the package in the future via NuGet.
```
<ItemGroup>
    <SDKReference Include="TestPlatform.Universal, Version=$(UnitTestPlatformVersion)" />
  </ItemGroup>
```

**Comment by Plac3hold3r** [comment]
I have made the change in the template as well. The next version will allow you to update via NuGet.

**Comment by Plac3hold3r** [comment]
Changes available in templates 1.4.0
